### PR TITLE
Tests for utils

### PR DESCRIPTION
**Pull request by isSerge**
a bunch of tests for utils

### CODE REVIEW

File: backend/src/test/utils.test.ts [hunk 1]
```diff
@@ -0,0 +1,156 @@
...
+      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+      0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0,
+      0, 0, 0, 0, 0, 0, 0, 0, 0, 41, 154, 201,
+      57, 234, 71, 34, 209, 252, 247, 244, 135, 54, 117, 4,
+      14, 188, 131, 209, 68, 98, 12, 118, 32, 109, 190, 183,
+      70, 140, 243, 12, 223, 3, 23, 10, 46, 117, 151, 183,
+      183, 227, 216, 76, 5, 57, 29, 19, 154, 98, 177, 87,
+      231, 135, 134, 216, 192, 130, 242, 157, 207, 76, 17, 19,
+      20, 0, 0, 0
+    ])
```
Comment (nazar-pc) [inline backend/src/test/utils.test.ts:48]: `Buffer.from('hex string', 'hex')` would be more readable though

Comment (nazar-pc) [reply backend/src/test/utils.test.ts:48]: Also more cases with justifications, extrinsics and log items would be nice, but this is a start nevertheless.

**Review by nazar-pc — APPROVED** [review]

File: backend/src/test/utils.test.ts [hunk 2]
```diff
@@ -154,3 +132,45 @@ tap.test('isInstanceOfSignedBlockJsonRpc util function', (t) => {
...
+    const hex = 'f17a9ca2e515a68c6e1b0315c5182d237580507c60487f5a6b75c703aa3e84ab79025a4cfd95c29f8db93119a11e012c3905767cc4b7e3a3a2f6bc72879e51a47cd4907510396823ad5299bd8b7e3d922e182f117032a0653a85f9ad3cb58044a08608066175726120614c41170000000005617572610101c4dd7f7d2a7521a91b821c44d0b8a628e4d804c0428b20a8e281aeaff575001cd55f4540db0fa78e5521b330d1a7ef9e8b922ca723097b4ab83e59a13fb44f0c0000';
+
+    const expected = Buffer.from(hex, 'hex');
+
+    t.same(result, expected);
+
+    t.end();
+  });
+
+  tap.test('blockToBinary should convert SignedBlockJsonRpc with justifications to Buffer')
```
Comment (nazar-pc) [inline backend/src/test/utils.test.ts:173]: Is this an empty test?

**Review by nazar-pc — APPROVED** [review]

File: backend/src/test/utils.test.ts (hunk 2 above)
Comment (isSerge) [reply backend/src/test/utils.test.ts:173]: it's a todo test

**Pull request merged**